typescript/web/src/components/export-button/export-modal/utils/get-dataset-name.ts

```diff
@@ -0,0 +1,21 @@
+import { ExportFormat } from "../../../../graphql-types/globalTypes";
+
+export const getDatasetName = ({
+  datasetSlug,
+  format,
+}: {
+  datasetSlug: string;
+  format: ExportFormat;
+}) => {
+  const dateObject = new Date();
+  const date = `${dateObject
+    .toLocaleDateString()
+    .split("/")
+    .reverse()
+    .join("-")}T${String(dateObject.getHours()).padStart(2, "0")}${String(
+    dateObject.getMinutes()
+  ).padStart(2, "0")}${String(dateObject.getSeconds()).padStart(2, "0")}`;
```

Comment:
I know that the code doesn't comes from you but these lines feel risky and complex.
If you have a few time for this, it could be nice to use some intermediary variables and eventually unit-test the `getDatasetExportName` function.